android/build.gradle

```diff
@@ -2,7 +2,7 @@ group 'com.apptentive.apptentive_flutter'
 version '1.0'
 
 buildscript {
-    ext.kotlin_version = '1.6.0'
+    ext.kotlin_version = '1.7.21'
     repositories {
         google()
         mavenCentral()
```

Comment:
Should add this block to test until 6.0.4 is released and available in mavenCentral()
maven {
    url "https://oss.sonatype.org/content/groups/staging/"
    credentials {
        username = "<USER NAME>"
        password = "<PASSWORD"
}